docs/manuals/core/config/data-node-config.md

```diff
@@ -772,4 +772,30 @@ The scope is `SCENARIO` (default value), and default data is provided.
     `Config.configure_in_memory_data_node()^` or the method `Config.configure_data_node()^`
     with parameter `storage_type="in_memory"`.
 
+# Default data node configuration
+
+By default, if you don't provide any information when configure a datanode (except for the mandatory _**id**_), the `Config.configure_data_node()^` method will return a _pickle_ data node configuration with `SCENARIO` scope.
+
+To override the default data node configuration, one can use the `Config.configure_default_data_node()^` method.
```

Comment:
"one can" -> "you can".
  That what you've used in the text so far.

Reply:
I updated the previous sentence to "if there is no information provided ...", and keep the "one can" to be consistent with the rest of this page.